usr/share/rear/prep/default/320_include_uefi_env.sh

```diff
@@ -48,8 +48,8 @@ if grep -qw efivars /proc/mounts ; then
     SYSFS_DIR_EFI_VARS=/sys/firmware/efi/efivars
 fi
 
-# Next step is case-sensitive checking /boot for case-insensitive /efi directory (we need it):
-test "$( find /boot -maxdepth 1 -iname efi -type d )" || return 0
+# Next step is case-sensitive checking /boot and /efi for case-insensitive /efi directory (we need it):
+test "$( find /boot /efi -maxdepth 1 -iname efi -type d )" || return 0
```

Comment:
Shouldn't that be `find /boot /` so that the `-iname` would match `/efi`?

Reply:
Depends on what we want to do here. On my system:
```
$ find /boot /efi/ -maxdepth 1 -iname efi -type d
/efi/
/efi/EFI
```
What does this code try to do? I think it trys to find the `EFI` dir in the root of the ESP, which is not `/efi`, but `/efi/EFI`
In that case I should maybe add `-mindepth 1`, so that if case /efi exists but is empty the test fails.